.github/workflows/test.yml

```diff
@@ -29,8 +29,8 @@ jobs:
       uses: actions/setup-node@v2
       with:
         node-version: ${{ matrix.node-version }}
-        cache: 'yarn'
-    - run: yarn install
-    - run: yarn lint
-    - run: yarn build
-    - run: yarn test
+        cache: 'npm'
+    - run: npm install
+    - run: npm run lint
+    - run: npm run build
+    - run: npm run test
```

Comment:
Why switch to npm and node 16?

Reply:
For Yarn to npm that's because I always miss the `yarn.lock` and did `npm install`...
I feel like there's no extra benefits on using Yarn anymore, performance seems similar.

For Node 16, it's because (1) newer npm version has better performance, (2) NodeJS is going fast, Node 14 already reached end-of-life (though Node 16 will also reach end-of-life at Sep 23, maybe we should use 18?)

Reply:
After rebase got tons of errors... Maybe I will switch back to Yarn🌚

Reply:
turns out we should stick with 16 first if we want to use this lib properly in our web apps